src/components/LocationPage/ChartBlock.tsx

```diff
@@ -40,55 +35,50 @@ function ChartBlock(props: {
 
   const hasMetric = projections.hasMetric(metric);
 
-  const metricDefinition = getMetricDefinition(metric);
-  const chartHeaderTooltip = metricDefinition.renderInfoTooltip();
-  const disclaimerContent = renderDisclaimer(region, metric, provenance);
+  const overrideDisclaimer = getOverrideDisclaimer(region, metric, provenance);
 
   const chartHeight = isMobile ? 280 : 400;
 
+  const shareButtonProps = {
+    chartIdentifier: metric,
+    region,
+    stats,
+    showEmbedButton: false,
+  };
+
+  const modalContent = getMetricModalContent(region, metric, provenance);
+
   return (
     <>
       <SectionHeader ref={props.chartRef}>
         {getMetricNameExtended(metric)}
-        <>{chartHeaderTooltip}</>
         {showBetaTag && <BetaTag>New</BetaTag>}
       </SectionHeader>
-      <ChartDescription>
-        {getMetricStatusText(metric, projections)}
-      </ChartDescription>
       {hasMetric && (
         <>
           <MetricChart
             metric={metric}
             projections={projections}
             height={chartHeight}
           />
-          <LocationPageSectionFooter>
-            <DisclaimerWrapper>{disclaimerContent}</DisclaimerWrapper>
-            <ShareButtons
-              chartIdentifier={metric}
-              region={region}
-              stats={stats}
-              showEmbedButton={false}
-            />
-          </LocationPageSectionFooter>
+          <MetricChartFooter
+            footerText={getMetricStatusText(metric, projections)}
+            overrideDisclaimer={overrideDisclaimer}
+            shareButtonProps={shareButtonProps}
+            modalContent={modalContent}
+            modalHeader={getMetricName(metric)}
+          />
```

Comment:
How about passing `region`, `projection` and `metric` as props to `MetricChartFooter` and calculating each part in that component instead? That would make the API of `MetricChartFooter` look a bit more like `MetricChart` and hide implementation details that are not relevant in `ChartBlock`